oh_staff_ui/management/commands/create_mods_records.py

```diff
@@ -7,29 +7,63 @@
 logger = logging.getLogger(__name__)
 
 
-def create_mods_records(item_id: int) -> None:
-    """Given an item_id write the related MODS record to public location"""
-    try:
-        pi = ProjectItem.objects.get(id=item_id)
-        OralHistoryMods(pi).write_mods_record()
-    except (ProjectItem.DoesNotExist) as e:
-        logger.error(e)
-        raise CommandError(f"ProjectItem with id {item_id} does not exist")
-
-
 class Command(BaseCommand):
     help = "Django management command to generate Oral History MODS records"
 
     def add_arguments(self, parser):
         parser.add_argument(
             "-i",
             "--item_id",
+            nargs="+",
             type=int,
-            required=True,
-            help="The id of the item to export mods record of",
+            required=False,
+            help="The id(s) of the item to export mods record of",
+        )
+        parser.add_argument(
+            "-b",
+            "--bulk",
+            type=str,
+            required=False,
+            choices=["series", "interview", "audio", "all"],
+            help="The type of items to bulk export, choices are: series, interview, assets and all",
         )
 
     def handle(self, *args, **options):
 
-        item_id = options["item_id"]
-        create_mods_records(item_id)
+        if options["item_id"]:
+            for item_id in options["item_id"]:
+                self._create_mods_record(item_id)
+
+        if options["bulk"]:
+            self._create_bulk_mods_records(options["bulk"])
+
+    def _create_mods_record(self, item_id: int) -> None:
+        """Given an item_id write the related MODS record to public location"""
+        try:
+            pi = ProjectItem.objects.get(id=item_id)
+            OralHistoryMods(pi).write_mods_record()
+            logger.info(f"Item: {pi.id} with ark {pi.ark} MODS record written")
```

Comment:
Logging is good... but when doing the whole set, it quickly fills up the log, which currently defaults to showing the 200 most recent lines.  If it's useful to know it succeeded, switch this to `logger.debug` so it will only be in our dev environments (once I finish my PR...).  However, I think logging the rare errors is enough, instead of the common successes - that's what I'm doing with my other logging cleanup.

Reply:
I am switching to `logger.debug` and will keep in, I think useful for debugging in local environment.